src/controllers/productsController.js

```diff
@@ -0,0 +1,17 @@
+const productsService = require('../services/productsService');
+
+const getAllProducts = async (req, res) => {
+  try {
+    const { limit, offset } = req.query;
```

Comment:
이런식으로 default를 줄 수 있습니다.
```suggestion
    const { limit=10, offset=0 } = req.query;
```

Reply:
옷 감사합니다! 프론트에서 6개씩 받아오면 좋겠다고 하셔서 6으로 넣어두겠습니다~